src/data/markdown/translated-guides/en/02 Using k6/12 Execution context variables.md

```diff
@@ -31,17 +31,24 @@ $ k6 run --vus 10 --iterations 100 script.js
 
 ## \_\_VU and \_\_ITER
 
-**\_\_VU** and **\_\_ITER** are both global variables with execution context information that k6 makes
-available to the test script.
+**\_\_VU** and **\_\_ITER** are both global variables with execution context information that k6 makes available to the test script.
 
 ### \_\_ITER
 
 A numeric counter with the current iteration number for a specific VU. Zero-based.
 
 ### \_\_VU
 
-Current VU number. The value is assigned incrementally for each new VU instance. One-based.
-However, VU number is 0 while executing the setup and teardown functions.
+Current VU number in use. The value is assigned incrementally for each new VU instance, starting from one. The variable will be 0 while executing the setup and teardown functions.
+
+It's important to remember that this value is shown _per server instance running you test_. How many servers we use depends on the size of your test and how you spread it across different load zones. So for a larger test, the value of `__VU` will likely not reflect the actual _total_ number of VUs in your test at a given moment.
+
+- For tests using **1-999** VUs per load zone, we use server instances that handle 300 VU's per machine. So for a 900 VU test using one load zone, there will be three instances and the value of `__VU` will vary from 1-300 per machine.
+- For tests using **1000-4001** VUs per load zone, we use server instances that handle 1200 VUs per machine. So for a 1000 VU, 1-load-zone test, your `__VU` value will go all the way to 1000.
+- For tests with **>4001** VUs per load zone, we use instances that handle 5000 VUs per machine.
```

Comment:
I would add a note that this is the *default* and we can change it so it might be different for clients with different needs.